src/rvya25-profile.adoc

```diff
@@ -0,0 +1,466 @@
+  those caused by execution of the `EBREAK` or `C.EBREAK` instructions.
+  For virtual-instruction and illegal-instruction exceptions, `stval` must be written with the
+  faulting instruction.
+
+- *Sscounterenw* For any `hpmcounter` that is not read-only zero, the
+  corresponding bit in `scounteren` must be writable.
+
+- *Svpbmt* Page-based memory types
+
+- *Svinval* Fine-grained address-translation cache invalidation.
+
+- *Svnapot* NAPOT translation contiguity.
+
+- *Sstc* supervisor-mode timer interrupts.
+
+- *Sscofpmf* count overflow and mode-based filtering.
+
+- *Ssu64xl* `sstatus.UXL` must be capable of holding the value 2
+(i.e., UXLEN=64 must be supported).
+
+- *Sha* The augmented hypervisor extension.
+
+NOTE: Sha is defined in the RVA23 profile document.
+
+The following mandatory extension is RVY25S64 specific:
+
+- *Svy* Read/write access control for capability tags in the page table entries.
+
+The following extension is mandatory in RVA23 but is _incompatible_ with RVY:
+
+- *Ssnpm* Pointer masking, with `senvcfg.PMM` and `henvcfg.PMM` supporting,
+at minimum, settings PMLEN=0 and PMLEN=7.
+
+NOTE: How to benefit from the combination of pointer masking and CHERI is a research topic.
+
+==== RVYA25S64 Optional Extensions
+
+===== Localized Options
+
+There are no privileged localized options in RVYA25S64.
+
+===== Development Options
+
+There privileged development options in RVYA25S64.
+
+- *Svucrg* Page-based revocation (preventing use-after-free in virtual memory systems).
+
+- *Svucrglct* (implies *Svucrg*) checking loaded capability tags before taking exceptions to optimize revocation.
+
+===== Expansion Options
+
+- *Sv48* Page-based 48-bit virtual-memory system.
+
+- *Sv57* Page-based 57-bit virtual-memory system.
+
+- *Zkr*  Entropy CSR.
+
+- *Svadu* Hardware A/D bit updates, and RVY revocation updates if Svucrg is implemented.
+
+- *Sdtrig* Debug triggers.
+
+- *Ssstrict* No non-conforming extensions are present.  Attempts to
+   execute unimplemented opcodes or access unimplemented CSRs in the
+   standard or reserved encoding spaces raises an illegal instruction
+   exception that results in a contained trap to the supervisor-mode
+   trap handler.
+
+NOTE: The RVA23 profile contains more details of Ssstrict.
+
+- *Svvptc* Transitions from invalid to valid PTEs will be visible in
+   bounded time without an explicit memory-management fence.
+
+The following extension is an expansion option in RVA23 but is _incompatible_ with RVY:
+
+- *Sspm* Supervisor-mode pointer masking, with the supervisor execution
+   environment providing a means to select PMLEN=0 and PMLEN=7 at minimum.
+
+NOTE: How to benefit from the combination of pointer masking and RVY is a research topic.
+
+===== Transitory Options
+
+The RYA25S64 hybrid options are transitory as over time it is expected that the compatibility requirement will disappear.
+
+- *Smyre* Support for enabling CHERI at different privilege levels (implies Zyhbrid)
```

Comment:
This should just be part of having Zyhybrid (which isn't spelled correctly here) and modes other than M-mode

Reply:
see https://github.com/riscv/riscv-cheri/pull/763